src/Core/MySQL/PacketsProtocolText.cpp

```diff
@@ -1,12 +1,13 @@
+#include <Columns/ColumnNullable.h>
```

Comment:
By convention, the corresponding header file of a source file (PacketsProtocolText.h) is included always first.